<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>604</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>QPar2</string>
  </property>
  <property name="dockNestingEnabled" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>7</hsizetype>
     <vsizetype>7</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="fileLabel" >
        <property name="text" >
         <string>Source File: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sourceFileEdit" />
      </item>
      <item>
       <widget class="QPushButton" name="browseButton" >
        <property name="text" >
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="tabPosition" >
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape" >
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <property name="elideMode" >
       <enum>Qt::ElideNone</enum>
      </property>
	  <widget class="QWidget" name="tabFiles" >
       <attribute name="title" >
        <string>Files</string>
       </attribute>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tableWidget" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>7</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode" >
           <enum>Qt::ElideLeft</enum>
          </property>
          <property name="showGrid" >
           <bool>false</bool>
          </property>
          <property name="gridStyle" >
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="columnCount" >
           <number>3</number>
          </property>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabInfo" >
       <attribute name="title" >
        <string>Informations</string>
       </attribute>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="lblSetId" >
            <property name="text" >
             <string>Set ID:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblBlockSize" >
            <property name="text" >
             <string>Block size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblChunkSize" >
            <property name="text" >
             <string>Chunk size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblDataBlock" >
            <property name="text" >
             <string>Data block:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblDataSize" >
            <property name="text" >
             <string>Data size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblRecoverable" >
            <property name="text" >
             <string>Recoverable files:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblOtherFiles" >
            <property name="text" >
             <string>Other files:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="globalProgressBar" >
      <property name="maximum" >
       <number>1000</number>
      </property>
      <property name="value" >
       <number>0</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="repairButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Repair</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="verifyButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Verify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton" >
        <property name="text" >
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statBar" />
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>496</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel" >
     <x>535</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
